packages/roosterjs-content-model-plugins/lib/imageEdit/utils/imageEditUtils.ts

```diff
@@ -0,0 +1,120 @@
+import { MIN_HEIGHT_WIDTH } from '../constants/constants';
+
+/**
+ * @internal
+ */
+export function getPx(value: number): string {
+    return value + 'px';
+}
+
+/**
+ * @internal
+ */
+export function isASmallImage(widthPx: number, heightPx: number): boolean {
```

Comment:
This function is duplicated in getHTMLImageOptions.ts, let keep only one of them